bridge.py

```diff
@@ -177,7 +182,7 @@ def get_parser():
         help='The entries to be ignored from the mapping file.',
         nargs='*',
         default=['IRIS:LR20:130:CONFG_SEL', 'LASR:IN20:475:PWR1H',
-                 'GUN:IN20:1:GN1_ADES'],
+                 'GUN:IN20:1:GN1_ADES','ACCL:IN20:300:L0A_ADES'],
```

Comment:
Poking this constant "variable" nukes the model. So it is now added into the denylist until we fix lume-model.